web/src/utils/iframes.ts

```diff
@@ -0,0 +1,189 @@
+import { TFunction } from 'i18next'
+
+import { CONSENT_ROUTE, ExternalSourcePermissions } from 'api-client'
+
+import { IFRAME_BLANK_SOURCE, IframeSources } from '../components/RemoteContent'
+import dimensions from '../constants/dimensions'
+
+export const EXTERNAL_SOURCES_COOKIE_NAME = 'Opt-In-External-Sources'
+export const addDoNotTrackParameter = (iframe: HTMLIFrameElement): void => {
+  if (iframe.src.includes('vimeo')) {
+    const url = new URL(iframe.src)
+    url.searchParams.append('dnt', '1')
+    iframe.setAttribute('src', url.href)
+  }
+}
+
+export const preserveIFrameSourcesFromContent = (
+  index: number,
+  source: string,
+  setExistingIframes: (sources: IframeSources) => void,
+  contentIframeSources: IframeSources,
+): void => {
+  const updatedContentSources: IframeSources = contentIframeSources
+  updatedContentSources[index] = source
+  setExistingIframes(updatedContentSources)
+}
+
+export const hideIframe = (iframe: HTMLIFrameElement): void => {
+  iframe.setAttribute('src', IFRAME_BLANK_SOURCE)
+  iframe.setAttribute('style', 'display:none')
```

Comment:
I don't love that we're only hiding the iframe instead of removing it.

Reply:
yes but you can't remove them, because you don't know where they were in the DOM.Thats why i replace the src and hide them.
Even by replacing the source, the iframe can't track anything or deliver data

Reply:
Huh, we can't get it via the `parentNode` and `removeChild`?

Reply:
no @LeandraH you remove the node with the iframe and its gone. We don't recreate the old DOM every time as we do in native. So we can not relocate where it was before